app/client/IDataFeed.js

```diff
@@ -0,0 +1,24 @@
+import abi from './IDataFeed.abi'
+import contractFn from './contractFn'
+
+const contractName = 'iDataFeed'
```

Comment:
might consider changing the name to `IDataFeed`

Reply:
was using the same pattern from futarchyApp